src/userobjects/MDRunBase.C

```diff
@@ -0,0 +1,124 @@
+/**********************************************************************/
+/*                     DO NOT MODIFY THIS HEADER                      */
+/* MAGPIE - Mesoscale Atomistic Glue Program for Integrated Execution */
+/*                                                                    */
+/*            Copyright 2017 Battelle Energy Alliance, LLC            */
+/*                        ALL RIGHTS RESERVED                         */
+/**********************************************************************/
+
+#include "MDRunBase.h"
+#include "MooseMesh.h"
+
+#include "libmesh/mesh_tools.h"
+
+// custom data load and data store methods for a class with virtual members (vtable pointer must not
```

Comment:
This comment is not appropriate here. We need the custom load/store because we have members with data on the heap.